client/my-sites/earn/memberships/index.jsx

```diff
@@ -259,6 +259,32 @@ class MembershipsSection extends Component {
 		);
 	}
 
+	renderOnboarding() {
+		return (
+			<div>
+				<SectionHeader label={ this.props.translate( 'About Recurring Payments' ) } />
+				<Card>
+					<div className="memberships__module-content module-content">
+						<p>
+							{ this.props.translate(
+								'Start collecting subscription payments! Recurring Payments is a feature inside the block editor. When editing a post or a page you can insert a button that will allow you to collect paying subscribers.'
+							) }
+						</p>
+					</div>
+				</Card>
+				<Notice
+					text={ this.props.translate( 'Read more about Recurring Payments feature' ) }
```

Comment:
should be "Read more about the Recurring Payments feature."

Reply:
fixed!